landing_page/lib/tasks/archive_posts.rake

```diff
@@ -0,0 +1,10 @@
+desc "Archive posts periodically based on if they are one week old or more"
+
+task archive: :environment do
+    archived_posts = Micropost.order(created_at: :desc).kept.where("created_at <= ?", Time.now-7.days)
```

Comment:
This variable name isn't semantically correct. The posts are not archived at this point. See suggestions below:

```suggestion
    posts_older_than_seven_days = Micropost.order(created_at: :desc).kept.where("created_at <= ?", Time.now-7.days)
```

```suggestion
    posts_to_be_archived = Micropost.order(created_at: :desc).kept.where("created_at <= ?", Time.now-7.days)
```

Reply:
Changed in commit 34a92c279f74b19cf3a746d379c6be6fd499e2cf